src/leap/common/events/signals.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# proto.py
+# Copyright (C) 2015 LEAP
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful",
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""
+Communication protocol definitions.
+"""
+
+
+from enum import Enum
```

Comment:
> and a new pyOpenSSL which needs enum34 

when packaging debs, we're not providing pyopenssl, the distro does.
and bundles are not a problem since the lib scope is self-contained